docs/usage/bazel.md

```diff
@@ -34,7 +174,7 @@ git_repository(
 
 Renovate uses the list of **tags** on the remote repository (GitHub) to detect a new version.
 
-## http_archive and http_file
+### `http_archive` and `http_file`
```

Comment:
Did you check if monospacing the heading at line 177 and 197 breaks any links? I don't know if we have links in the first place, but you should check.